harness_suite.go

```diff
@@ -0,0 +1,93 @@
+package poplar
+
+import (
+	"context"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/mongodb/grip"
+)
+
+// BenchmarkSuite is a convenience wrapper around a group of suites.
+//
+// You can use the Standard() function to convert these benchmarks to
+// a standard go library benchmark function.
+type BenchmarkSuite []*BenchmarkCase
```

Comment:
Does this need to be a slice of pointers? They tend to lead to `range` bugs.

Reply:
Yeah, I think it's alright here (famous last words) given how we're using the slice. And  ~required to support the fluent interface, given that the items need to be mutable.